mitodl/mit-open
# Adds learner testimonials component for interior pages

### What are the relevant tickets?

Closes mitodl/hq#4457

### Description (What does it do?)

This adds a component that can be included on any page to display testimonials, and then adds that to the `FieldPage` component so it should display on the field search pages (which includes offeror pages, I think).

### Screenshots (if appropriate):

Screenshots are of the component on the homepage since my local search is broken. 

Desktop:

![image](https://github.com/mitodl/mit-open/assets/945611/c0b6b7d0-8b29-4d0f-a963-c2f2a6a39966)


Mobile:

![image](https://github.com/mitodl/mit-open/assets/945611/cc7bdb80-06ce-4011-8420-f62843c98562)


### How can this be tested?

You will need to configure your app with testimonial data. This can be anything; you just need records in there.

You should see the testimonial card on the offeror pages - for example, http://od.odl.local:8063/c/offeror/ocw/ . You should only see testimonials that are attached to that particular offeror. You should be able to paginate through them, and the design should match the design in Figma. 

You should also see the testimonial card on the search pages as well; there, it should show all the testimonials. 

### Additional Context

- I added a bit of margin on top of the pager buttons - this isn't in the Figma but they were sort of getting stuck too close to the name block.
- You can also test this by adding the testimonial card to the homepage (which is what I did for testing, since my search is broken). Look in `mit-open/src/pages/FieldPage/FieldPage.tsx` to see how but you really just have to drop the component in anywhere in there.
